### PR TITLE
Created doc: GROUP BY

### PR DESCRIPTION
**Pull request by randomJoe211**
Created doc GROUP BY and updated aggregate functions

### CODE REVIEW

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md [hunk 1]
```diff
@@ -29,8 +29,17 @@ int rand32(int min, int max) | Returns a random 32 bit integer in [min, max).|
...
+max() | Returns the maximum value.|
+min() | Returns the minimum value.|
+std() | Returns the population standard deviation.|
+sum() | Returns the sum value.|
+bit_and() | Bitwise AND.|
+bit_or() | Bitwise OR.|
+bit_xor() | Bitwise exclusive OR (XOR).|
+int size() | Returns the number of elements in a list.|
+map(fun, iter) | Returns a map object after applying the given function to each item of a given iterable.|
+int range(int, int, step) | Returns a list of integers from the starting integer to the end integer in optional steps.|The default step is one.|
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:45]: [min, max] or [min, max)  cypher is the former style,and C++ is the later sytle

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md [hunk 2]
```diff
@@ -29,8 +29,17 @@ int rand32(int min, int max) | Returns a random 32 bit integer in [min, max).|
...
+avg() | Returns the average value of the argument.|
+count() | Returns the number of records.|
+max() | Returns the maximum value.|
+min() | Returns the minimum value.|
+std() | Returns the population standard deviation.|
+sum() | Returns the sum value.|
+bit_and() | Bitwise AND.|
+bit_or() | Bitwise OR.|
+bit_xor() | Bitwise exclusive OR (XOR).|
+int size() | Returns the number of elements in a list.|
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:43]: can size() or length() work on a map?

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md (hunk 1 above)
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:45]: `|The default step is one.|` 
The format is not right here

File: mkdocs.yml [hunk 3]
```diff
@@ -127,6 +127,7 @@ nav:
           - ORDER BY: 3.ngql-guide/8.clauses-and-options/order-by.md
           - LIMIT: 3.ngql-guide/8.clauses-and-options/limit.md
           - SKIP: 3.ngql-guide/8.clauses-and-options/skip.md
+          - GROUP BY: 3.ngql-guide/8.clauses-and-options/group-by.md
```
Comment (Amber1990Zhang) [inline mkdocs.yml:130]: nice

File: docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
++---------------------+------------+
+| "Manu Ginobili"     | 2          |
++---------------------+------------+
+| "Dejounte Murray"   | 1          |
++---------------------+------------+
+| "Danny Green"       | 1          |
++---------------------+------------+
+| "Tony Parker"       | 2          |
++---------------------+------------+
+| "Shaquille O'Neal"   | 1          |
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md:34]: why this line not aligned? Could u pls adjust it manually?

File: docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+| "LaMarcus Aldridge" | 1          |
++---------------------+------------+
+| "Marco Belinelli"   | 1          |
++---------------------+------------+
+Got 10 rows (time spent 3527/4423 us)
+```
+
+## Group and sort the output
+
+The following statement finds the follower number of players, groups the output by player names, and sorts the output by the follower number.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md:45]: amazing example!

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:45]: > [min, max] or [min, max) cypher is the former style,and C++ is the later sytle

OpenCypher has the same syntax: range(start, end [, step]).
image

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md (hunk 2 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:43]: size() can but length() can't. I'll modify the doc.

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md [hunk 6]
```diff
@@ -29,8 +29,17 @@ int rand32(int min, int max) | Returns a random 32 bit integer in [min, max).|
...
+map(fun, iter) | Returns a map object after applying the given function to each item of a given iterable.|
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:44]: amazing

**Review by whitewum — APPROVED** [review]

File: docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md (hunk 4 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md:34]: Good catch!

File: docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md (hunk 5 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/8.clauses-and-options/group-by.md:45]: Thank you for the extra thumbs up.

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md (hunk 1 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/6.functions-and-expressions/1.math.md:45]: I get your point now... And I've added `inclusive` in the doc.

**Review by Amber1990Zhang — APPROVED** [review]
LGTM

**Review by whitewum — APPROVED** [review]

**Pull request merged**